android/KMEA/app/src/main/java/com/tavultesoft/kmea/JSONParser.java

```diff
@@ -5,6 +5,9 @@
 package com.tavultesoft.kmea;
```

Comment:
You know... not that this needs to be done right now, but this feels like a good candidate for the `util` subpackage.